ecs/awsResources.go

```diff
@@ -0,0 +1,278 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package ecs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/awslabs/goformation/v4/cloudformation/ec2"
+	"github.com/awslabs/goformation/v4/cloudformation/elasticloadbalancingv2"
+
+	"github.com/awslabs/goformation/v4/cloudformation"
+	"github.com/awslabs/goformation/v4/cloudformation/ecs"
+	"github.com/compose-spec/compose-go/types"
+	"github.com/sirupsen/logrus"
+)
+
+// awsResources hold the AWS component being used or created to support services definition
+type awsResources struct {
+	sdk              sdk
```

Comment:
maybe a detail, but it might be nice to separate sdk from all other fields here (have an sdk field, and another field holding all AWS resources that are populated by `awsResources` parsing methods.) 
not required, but my first impression is it would regroup things in a clearer way, and maybe make it more obvious to add unit tests, or see at a glance what structs we need to populate. Sdk seems an outsider here